apps/installer/includes/os_configs/ubuntu.sh

```diff
@@ -12,8 +12,8 @@ sudo apt update
 # shared deps
 sudo apt-get -y install make cmake clang curl unzip libmysqlclient-dev libace-dev ccache google-perftools
 
-# Insstall boost 1.73 from ppa:mhier/libboost-latest for all os versions
-sudo apt-get -y install libboost1.73-dev
+# Insstall boost 1.74 from ppa:mhier/libboost-latest for all os versions
+sudo apt-get -y install libboost1.74-dev
```

Comment:
better :)

Reply:
okay okay :D
I install `1.76` now
`CMake` support max `1.75` in version `20.2`